selfdrive/car/hyundai/carcontroller.py

```diff
@@ -96,7 +96,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
 
 
     # 20 Hz LFA MFA message
-    if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE]:
+    if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE, CAR.SANTA_FE]:
```

Comment:
Would this be a problem for the 2019 Santa FE that doesn't have LFA? Otherwise we need to split them into two different cars. Maybe you can find someone on discord with a 2019 one and have them try this PR.

Reply:
2019 and 2020 are same platform

Reply:
Any vehicle that steers down to 0 has LFA supported.

Reply:
I test drove a 2019 Santa Fe and it has the same displays. I believe the only thing that is different between the 2019 and 2020 is the fingerprint.